packages/yew/src/virtual_dom/key.rs

```diff
@@ -81,10 +81,10 @@ mod test {
     #[test]
     fn all_key_conversions() {
         html! {
-            <key="string literal">
+            <div key="string literal">
```

Comment:
Are there no fragments with keys anymore or what's this about?

Reply:
I think it's the list thingy `<> .... </>` which you can have a key on

Reply:
It's a change I forgot to revert. Fragments can have keys, I'll fix that